web/src/views/FileBrowserView/PublishFileBrowser.vue

```diff
@@ -65,6 +65,49 @@
               </v-icon>
               {{ item.name }}
               <v-spacer />
+
+              <v-list-item-action>
```

Comment:
This button still renders when you are not logged, and presumably when you aren't an admin. This should have a `v-if` that checks if `@rest.user` is in the owners list for the current dandiset, or if the user is an admin.

Reply:
Accomplished via 5ed937bbe7b6460782599fbc889bfde9dded6a51 and a couple of other commits.